dashboard/src/components/Catalog/Catalog.test.tsx

```diff
@@ -10,17 +11,28 @@ import SearchFilter from "../SearchFilter/SearchFilter";
 import Catalog, { filterNames } from "./Catalog";
 
 const defaultChartState = {
+  status: actions.charts.unstartedStatus,
```

Comment:
`unstartedStatus` sounds odd. Is the purpose to be able to always have a definite status? (otherwise I'd suggest using null initially for the status until it's been fetched?).

Reply:
Yes, it was initially required in the condition to render either ` renderEmptyCatalog()` or `renderCatalogContent()`. After checking the status, `renderEmptyCatalog` should be rendered if there are no results AND it is in `finishedStatus`.
So, as you pointed out, the `unstartedStatus` is no longer necessary. Removing it.